constraint/pkg/client/drivers/local/driver.go

```diff
@@ -72,6 +73,20 @@ type Driver struct {
 	clientCertWatcher *certwatcher.CertWatcher
 }
 
+// regoResultMeta has rego specific metadata for a result.
+type regoResultMeta struct {
+	// templateRunTime is the number of milliseconds it took to evaluate all constraints for a template.
+	templateRunTime float64
+	// engineType is an engine set string for the kind of underlying engine that was used for a client.Review() call.
```

Comment:
Given that this is `regoResultMeta`, we can assume engineType will always be `Rego`, right?